.env_example

```diff
@@ -1,4 +1,9 @@
 OPENAI_API_TYPE = "<your-api-type>"
 OPENAI_API_BASE = "https://<your-endpoint>.cognitiveservices.azure.com/"
 OPENAI_API_KEY = "<your-key>"
-OPENAI_API_VERSION = "<your-api-version>"
\ No newline at end of file
+OPENAI_API_VERSION = "<your-api-version>"
+
+#Spotify API
+CLIENT_ID = "your-client-id"
```

Comment:
let's try to be more specific (so that we can scale to othe APIs later).  E.g. `CLIENT_ID` for spotify should be `SPOTIFY_CLIENT_ID`